packages/build-info/src/file-system.ts

```diff
@@ -157,12 +157,11 @@ export abstract class FileSystem {
 
     if (absoluteTo.startsWith(absoluteFrom)) {
       // lazily matches a slash afterwards if it's a directory
-      return absoluteTo.replace(new RegExp(`^${absoluteFrom}/*`), '')
+      return absoluteTo.substring(absoluteFrom.length).replace(/^\//, '')
     }
 
-    // split by / excluding the starting slash
-    const fromParts = this.join(absoluteFrom).split(/(?<!^)\//gm)
-    const toParts = this.join(absoluteTo).split(/(?<!^)\//gm)
+    const fromParts = join(absoluteFrom).split('/')
+    const toParts = join(absoluteTo).split('/')
```

Comment:
What is the difference between `join` and `this.join`?

Reply:
oh this was a mistake `join` is the platform-independent implementation above in this file. And `this.join` will be using the version that is available on the platform (node would be `path.join` and on the web it will be the platform independent one.

I've updated with a commit.